typesafe/src/main/scala/com/github/fit51/reactiveconfig/typesafe/TypesafeUtils.scala

```diff
@@ -0,0 +1,74 @@
+package com.github.fit51.reactiveconfig.typesafe
+
+import java.nio.file.Path
+import java.util
+
+import com.github.fit51.reactiveconfig.Value
+import com.github.fit51.reactiveconfig.parser.ConfigParser
+import com.github.fit51.reactiveconfig.typeclasses.Effect
+import com.typesafe.config._
+
+import scala.collection.JavaConverters._
```

Comment:
use `import scala.jdk.CollectionConverters._`

Reply:
It breaks compilation of scala2.12 modules.